cmake/Platform/Common/Install_common.cmake

```diff
@@ -170,6 +182,23 @@ function(ly_setup_target OUTPUT_CONFIGURED_TARGET ALIAS_TARGET_NAME absolute_tar
     list(REMOVE_DUPLICATES INTERFACE_BUILD_DEPENDENCIES_PLACEHOLDER)
     string(REPLACE ";" "\n" INTERFACE_BUILD_DEPENDENCIES_PLACEHOLDER "${INTERFACE_BUILD_DEPENDENCIES_PLACEHOLDER}")
 
+    # If the target is an executable/application, add a custom target so we can debug the target in project-centric workflow
+    if(should_create_helper)
+        string(REPLACE ".Imported" "" RUN_TARGET_NAME ${NAME_PLACEHOLDER})
+        set(target_types_with_debugging_helper EXECUTABLE APPLICATION)
+        if(NOT target_type IN_LIST target_types_with_debugging_helper)
+            message(FATAL_ERROR "Cannot generate a RUN target for ${TARGET_NAME}, type is ${target_type}")
+        endif()
+        set(TARGET_RUN_HELPER
+"add_custom_target(${RUN_TARGET_NAME})
+set_target_properties(${RUN_TARGET_NAME} PROPERTIES 
+    FOLDER \"CMakePredefinedTargets/SDK\"
```

Comment:
I just realized this is "hidden" from the Users when they open up a newly generated project centric build solution.

![image](https://user-images.githubusercontent.com/56135373/127576184-30a5299d-6ce2-451d-908e-c6848cc53b1c.png)

It is doesn't pop out to the user that there is a target for the SDK available in the Solution, until they expand the CMakePredefinedTargets..

Reply:
I think the state of expansion in the folders is saved in the user file, the first time everything should be expanded

Reply:
That is from me creating that Solution from a non-existent build directory.

Reply:
I would be ok putting it at the root `SDK`, or `O3DE_SDK` or `O3DE`
(will do it in my next change since this is already done in development)

Reply:
Good with me.